examples/add-receiver-creator/rendered_manifests/configmap-agent.yaml

```diff
@@ -222,8 +222,11 @@ data:
             config:
               extraDimensions:
                 metric_source: kubernetes-scheduler
-              port: 10251
+              port: 10259
+              skipVerify: true
               type: kubernetes-scheduler
+              useHTTPS: true
+              useServiceAccount: true
```

Comment:
Can you explain why we're also adding the `skipVerify`, `useHTTPS`, and `useServiceAccount` key-value pairs in this change?

Reply:
They have [secured the metrics endpoint of kube-scheduler](https://github.com/kubernetes/kubernetes/pull/96345), so we need to authenticate with k8s service account when scraping `/metrics` from scheduler.

Reply:
If I recall properly, w need [skipVerify](https://github.com/signalfx/signalfx-agent/blob/a963c180611efabc6cb8f8e3abb3f3584b7d59a6/docs/monitors/kubernetes-scheduler.md?plain=1#L38)=true because the certificates for authentication are usually self hosted in a Kubernetes cluster.